install_files/ansible-base/securedrop-prod.yml

```diff
@@ -2,6 +2,8 @@
 ---
```

Comment:
Would it make sense to also apply these changes to `securedrop-staging.yml`?

Reply:
Sure.

Reply:
Summarizing out of band discussion, we should _also_ add the locale blocks to the other "prod" SD playbooks, such as `-logs` and `-backup`. Technically only `-logs` uses `register`, which is how we first encountered the error we're aiming to resolve here, but adding coverage to the other playbooks will also let us broaden the config test and reuse the existing "check all prod SD playbooks" logic.

I'll tack on a commit to implement this before final review.